lib/matplotlib/ticker.py

```diff
@@ -973,6 +1078,87 @@ def format_eng(self, num):
         return formatted.strip()
 
 
+class PercentFormatter(Formatter):
+    """
+    Format numbers as a percentage.
+
+    How the number is converted into a percentage is determined by the
+    `mx` parameter. `mx` is the data value that corresponds to 100%.
+    Percentages are computed as ``x / mx * 100``. So if the data is
+    already scaled to be percentages, `mx` will be 100. Another common
+    situation is where `max` is 1.0.
+    """
+    def __init__(self, mx=100, decimals=None, symbol='%'):
```

Comment:
I think people will rarely want to specify `max`, so I would put it after the `decimals` kwarg.